jinahub/indexers/searcher/compound/FaissPostgresIndexer/README.md

```diff
@@ -61,14 +61,26 @@ with get_my_flow() as flow:
     flow.post(
         on='/sync',
         # we need some specific parameters here
+        # startup is only needed the 1st time, to actually
+        # create the FaissSearcher
```

Comment:
Sorry for being picky here, but from an APIs perspective these calls are strange. I know why we have the startup argument historically, but I believe we can simplify.

Let's look at the `_sync_only_delta` function (which we are talking about here essentially). We now use the very same function on `_kv_indexer` side. The only difference is, that we handle it differently to the `__init__` of Faiss and the `_add_delta` (which by the way should not be a private function, since we access it from the FaissPostgresIndexer). We could also always initialize a FaissSearcher when the FaissPostgresIndexer is initiallized and then only do `_add_delta` call. This would remove the `startup` parameter and make the user interface easier. Or do I miss some important point?

Since we want the delta update to be the default update, we can default `only_delta` to `True` and don't need any parameter on this call. That would be even more elegant :)

Reply:
> We could also always initialize a FaissSearcher when the FaissPostgresIndexer is initiallized and then only do _add_delta call. 

Alternatively, we can check on each `_sync_only_delta` call, if a FaissSearcher exists. If not, create one.

Reply:
>  remove the startup parameter and make the user interface easier.

I like this idea. Especially for the k8s deployment, it is impossible to determine whether it is the first time.

Reply:
Refactored. Check usage in `test_start_up`